introduction.md

```diff
@@ -0,0 +1 @@
+This is the introduction file. Now I just gotta learn MarkDown.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.